samples/remote_host_attestation/README.md

```diff
@@ -0,0 +1,128 @@
+
+# The host verification sample
+
+- Demonstrates the attestation of a remote host
+- Requires only the `openenclave/host_verify.h` header, as well as some C standard header files
+- Also runs in simulation mode and when both BUILD_ENCLAVES and HAS_QUOTE_PROVIDER flags are turned off
+
+Prerequisites: you may want to read [Common Sample Information](../README.md#common-sample-information) before going further
+
+## About the host verification sample
+
+When a host is doing host verification, or remote host attestation to be exact, it means that it is trying to authenticate its hardware and software settings so that it can determine whether or not to trust the remote host.
+This sample does remote host attestation by taking two arguments:
+
+- An SGX report
+- An SGX certificate
+
+However, the settings vary from time to time, so here we do not provide a pair of sample report and certificate, where a user can generate them with `oecert`. See [here](../../tests/tools/oecert/README.md) for more details.
```

Comment:
>../../tests/tools/oecert/README.md [](start = 170, length = 34)

Developers may read this readme.md from installed host_verify package. Suggest using absolute path (i.e. [https://github.com/openenclave/openenclave/](https://github.com/openenclave/openenclave/)...).